math/convertall/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 ConvertAll 0.8.0 v
+github.tarball_from releases
+
+python.versions     34 35 36 37 38 39
+python.default_version     38
+
+qt5.min_version     5.4
+
+name                convertall
+version             0.8.0
+license             GPL-2+
+categories          math science
+platforms           darwin
+maintainers         {@khorton kilohotel.com:kevin01}
```

Comment:
if you add ```openmaintainer``` you allow others to make minor changes to things. We usually do that unless it's a critical port of some kind.